### ISSUE TITLE
Feature Request: Quicker ways to get links into Helium

### ISSUE THREAD

**Issue by PuzzleHeaded555** [JadenGeller/Helium#250]
- When first open, display the "Open link" / "open file" combo window
- Update the current "Open Link", so the link text box is the first selected and that hitting return on the keyboard loads it.
- How about no need for the "Open Link" window, just have the ability to past right in it or have it pick up from the past board (if a video isn't currently playing)
- Also, Help button in the menu should go to the website or something.

**Comment by slashlos** [comment]
Helium 3 is pasteboard sensitive with the URL dialog

**Comment by ianshowell** [comment]
+1 to auto-selecting the link box upon pressing cmd+l or the like for easy pasting.

**Comment by proactivematter** [comment]
Yes... not being able to auto paste, or, at the very list, have the link box selected severely limits the workflow.
